test/unit-luatest/storage_exports_test.lua

```diff
@@ -135,3 +135,16 @@ test_group.test_deploy_privs = function(g)
         box.schema.role.drop('two')
```

Comment:
In the commit message: `exports_deply_funcs` -> `exports_deploy_funcs`.

Reply:
Fixed